### ISSUE TITLE
Well Edit - Cannot change a value to unspecified

### ISSUE THREAD

**Issue by nathanbollig** [nathanbollig/rt-quic-db#106]
Currently the user can change a field in well edit to an allowed value, but blanking out the field (to change it from its value to empty) has no effect. For example, if a well record has the sample unspecified, once a sample has been specified, it cannot be changed back to unspecified.

This behavior was selected originally because some fields load as empty if multiple wells are selected and there are multiple values across the selected wells for that field. If a value is not specified in that field, then the form submits with blank. From the perspective of the form submit, this case is indistinguishable from the case where the user deliberated submitted with a blank value.

In summary, there are two cases that need to be handled:
 - Multiple wells are selected and for a given field those wells have different values, so the field loads as blank and when submitted will be blank in that field. Desired behavior: do nothing (currently works).
 - Regardless of the number of wells selected or their current values in a field, the user deliberately blanks out a field and intends for all records to be blank in that field. Desired behavior: change all selected records in that field (currently doesn't happen because of behavior implemented in first bullet).

Unsure of how to handle this conflict in behavior.

**Comment by nathanbollig** [comment]
Consider a form element that tracks whether a field was blanked. Use that (rather than value of the field itself) to signal blanking of the field in the selected Well Condition records.
